.spelling

```diff
@@ -212,13 +213,21 @@ pprof
 pre-commit
 pytorch
 qps
+randAlpha
```

Comment:
This block shouldn't be needed as you've backticked the words in the docs.

Reply:
Ah, right. Fixed!